meetings/2023-04-05.md

```diff
@@ -47,23 +54,99 @@
+* SC: PAC, may I add this suggestions and assign you to merge? 
+* PAC: Yes. 
+* TT: This is calling it storage as a singular. Makes sense to keep singular everywhere else. 
+* SC: Yes. 
+
+ACTION: PAC to merge.
+
+* AD: *leaves meeting, sends regrets on chat*
 
 ### Update scope - remove client-client
 URL: https://github.com/solid/solid-wg-charter/pull/26
 
+[More convenient diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fsolid-wg-charter%2Ffix%2Fcharter-drafts-template%2Fcharter%2Findex.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fsolid-wg-charter%2Ffix%2Fscope-remove-client-client%2Fcharter%2Findex.html)
+
 * SC: Resolves https://github.com/solid/solid-wg-charter/issues/23
 * SC: Part of https://github.com/solid/solid-wg-charter/issues/9
+* SC: Part of a series of considerations around scope (issue #9). Original call for WG was focusing on Solid Protocol which is largely server-client. 
+* SC: We have PAC and eP approval. Assigning to PAC. Questions or objections? 
+* SC: No objections. 
+* JZ: Yes, client-client should be on its own not part of the WG.
+
+ACTION: PAC to merge [pull#26](https://github.com/solid/solid-wg-charter/pull/26).
+ACTION: TT to make further suggestions via PR following merge of [pull#26](https://github.com/solid/solid-wg-charter/pull/26)
 
+* PAC: Will do those after next meeting. Need to leave now.
+* PAC: *leaves meeting*
 
 ### Scope needs to be tightly defined with narrow focus
 URL: https://github.com/solid/solid-wg-charter/issues/9
+* SC: We have one agreements in the community about this issue, having tightly defined scope. Previous topic w(PR) was about that. 
+* SC: When I created this issue the scope seemed to be everything we're doing in the CG, or anything that could potentially fall under Solid Pods. That felt too broad, considering the focus was on Solid Protocol. There could be best practice guidelines which are still under development or have not been incubated in the CG. More interestingly, the part suggested to be passed as BP is out of scope for the WG. 
+* SC: I read "recommend a set of practices" as "best practices or guidelines". If it were requirements it'd be a deliverable, but this seems to be BP&G. 
+* SC: In the Charter "out of scope" has two parts: definition of identify mechanisms and definition of linked data formats. The part about set of practices is a very broad thing being thrown in. I'm not sure if we need to recommend best practices around AuthN or identity where, for example, we're also thinking those are out of scope. If we do anything about identity mechanisms, we could do best practices, but this seems to be reaching to far.
+* eP: I'm not sure if the intention wasn't to somehow mention that we'd try to build on other standards. I guess it might have been formulated in this way to have flexibility around normative requirements or recommendations but not committing to requirements. 
```

Comment:
```suggestion
* SC: In the Charter, "out of scope" has two parts: definition of identify mechanisms, and definition of linked data formats. The part about set of practices is a very broad thing being thrown in. I'm not sure if we need to recommend best practices around AuthN or identity where, for example, we're also thinking those are out of scope. If we do anything about identity mechanisms, we could do best practices, but this seems to be reaching too far.
* eP: I'm not sure whether the intention wasn't to somehow mention that we'd try to build on other standards. I guess it might have been formulated in this way to have flexibility around normative requirements or recommendations, but not committing to requirements. 
```